api/test/e2e/route_export_test.go

```diff
@@ -0,0 +1,716 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package e2e
+
+import (
+	"net/http"
+	"strings"
+	"testing"
+)
+
+func TestRoute_Export(t *testing.T) {
+	exportStrR1 := `
+			"/hello_": {
+				"get": {
+					"operationId": "aaaaGet",
+					"requestBody": {},
+					"responses": {
+						"default": {
+							"description": ""
+						}
+					},
+					"security": [],
+					"x-apisix-enableWebsocket": false,
+					"x-apisix-hosts": ["foo.com", "*.bar.com"],
+					"x-apisix-labels": {
+						"build": "16",
+						"env": "production",
+						"version": "v2"
+					},
+					"x-apisix-plugins": {
+						"limit-count": {
+							"count": 2,
+							"key": "remote_addr",
+							"rejected_code": 503,
+							"time_window": 60
+						}
+					},
+					"x-apisix-priority": 0,
+					"x-apisix-status": 1,
+					"x-apisix-upstream": {
+						"nodes": {
+							"172.16.238.20:1980": 1
+						},
+						"type": "roundrobin"
+					}
+				},
+				"post": {
+					"operationId": "aaaaPost",
+					"requestBody": {},
+					"responses": {
+						"default": {
+							"description": ""
+						}
+					},
+					"security": [],
+					"x-apisix-enableWebsocket": false,
+					"x-apisix-hosts": ["foo.com", "*.bar.com"],
+					"x-apisix-labels": {
+						"build": "16",
+						"env": "production",
+						"version": "v2"
+					},
+					"x-apisix-plugins": {
+						"limit-count": {
+							"count": 2,
+							"key": "remote_addr",
+							"rejected_code": 503,
+							"time_window": 60
+						}
+					},
+					"x-apisix-priority": 0,
+					"x-apisix-status": 1,
+					"x-apisix-upstream": {
+						"nodes": {
+							"172.16.238.20:1980": 1
+						},
+						"type": "roundrobin"
+					}
+				}
+			}`
+	exportStrR1 = replaceStr(exportStrR1)
+
+	tests := []HttpTestCase{
+		{
+			Desc:         "hit route that not exist",
+			Object:       APISIXExpect(t),
+			Method:       http.MethodGet,
+			Path:         "/hello_",
+			Headers:      map[string]string{"Host": "foo.com"},
+			ExpectStatus: http.StatusNotFound,
+			ExpectBody:   "{\"error_msg\":\"404 Route Not Found\"}\n",
+		},
+		{
+			Desc:   "create route",
```

Comment:
it would be better to change the test case desc
```suggestion
			Desc:   "create route with uris and hosts",
```

Reply:
done.